Disable log file. Fixes a symlink attack vulnerability. See Debian Bug#504680.

diff -Nur -x '*.orig' -x '*~' yzis-1.0~alpha1/libyzis/debug.cpp yzis-1.0~alpha1.new/libyzis/debug.cpp
--- yzis-1.0~alpha1/libyzis/debug.cpp	2008-11-06 20:54:27.000000000 +0100
+++ yzis-1.0~alpha1.new/libyzis/debug.cpp	2008-11-06 20:56:47.000000000 +0100
@@ -112,14 +112,17 @@
 #ifdef DEBUG
     _level = YZ_DEBUG_LEVEL;
 #else
-    _level = YZ_WARNING_LEVEL;
+    _level = YZ_FATAL_LEVEL;
 #endif
 
+    /*
 #ifndef YZIS_WIN32_GCC
     setDebugOutput( "/tmp/yzisdebug-" + QString(getpwuid(geteuid())->pw_name) + ".log" );
 #else
     setDebugOutput( "/tmp/yzisdebug.log" );
 #endif
+    */
+    setDebugOutput( "stderr" );
 
     // our message handler does not manage to display all messages. So,
     // it is better left off disabled at the moment. The last one gets lost
